src/main/java/lingogo/logic/commands/ListCommand.java

```diff
@@ -3,25 +3,68 @@
 import static java.util.Objects.requireNonNull;
 import static lingogo.model.Model.PREDICATE_SHOW_ALL_FLASHCARDS;
 
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+import lingogo.commons.core.index.Index;
+import lingogo.logic.parser.ParserUtil;
+import lingogo.logic.parser.exceptions.ParseException;
 import lingogo.model.Model;
+import lingogo.model.flashcard.Flashcard;
+import lingogo.model.flashcard.FlashcardInGivenFlashcardListPredicate;
 
 /**
- * Lists all persons in the flashcard app to the user.
+ * Lists flashcards in the flashcard app to the user.
  */
 public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
     public static final String COMMAND_DESCRIPTION = "Lists all flashcards";
-    public static final String COMMAND_USAGE = "list";
-    public static final String COMMAND_EXAMPLES = "list";
+    public static final String COMMAND_USAGE = "list [NUMBER_OF_FLASHCARDS]";
+    public static final String COMMAND_EXAMPLES = "list\nlist 4";
 
     public static final String MESSAGE_SUCCESS = "Listed all flashcards";
 
+    private static String args;
+
+    public ListCommand() {
+        args = "";
+    }
+
+    public ListCommand(String userInput) {
+        args = userInput;
+    }
 
     @Override
-    public CommandResult execute(Model model) {
+    public CommandResult execute(Model model) throws ParseException {
```

Comment:
`execute` should only throw a `CommandException`. `ParseException` should only be used by the `CommandParser` classes